packages/workspace/src/generators/convert-to-monorepo/convert-to-monorepo.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  convertNxGenerator,
+  getProjects,
+  joinPathFragments,
+  ProjectConfiguration,
+  readNxJson,
+  Tree,
+  updateNxJson,
+} from '@nx/devkit';
+import { moveGenerator } from '../move/move';
+import { Schema } from './schema';
+
+export async function monorepoGenerator(tree: Tree, options: Schema) {
+  const projects = getProjects(tree);
+  mayebExtractTsConfigBase(tree);
+  await maybeExtractJestConfigBase(tree);
```

Comment:
All of the extract, should go into `move`. Regardless of if we are moving from a monorepo or not, if a user moves the root app, we should do this stuff.

This could be refactored more later.